remix-gql-tailwind/remix.config.js

```diff
@@ -1,11 +1,25 @@
+const path = require("path");
+
 /**
- * @type {import('@remix-run/dev/config').AppConfig}
+ * @type {import('@remix-run/dev').AppConfig}
  */
 module.exports = {
   appDirectory: "app",
+  cacheDirectory: "./node_modules/.cache/remix",
   assetsBuildDirectory: "public/build",
-  publicPath: "/build/",
-  serverBuildDirectory: "build",
-  devServerPort: 8002,
-  ignoredRouteFiles: [".*"],
+  publicPath: "/_static/build/",
```

Comment:
publicPath need to be changed like this for the arc deploy setup?

Reply:
Yes, at least based on the grunge stack and docs I've seen.